### PR TITLE
Chapter 14

### PR DESCRIPTION
**Pull request by balqis277**
ready

### CODE REVIEW

**Comment by balqis277** [comment]
ready

File: app/assets/stylesheets/relationships.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Relationships controller here.
```
Comment (huongnt-2545) [inline app/assets/stylesheets/relationships.scss:1]: delete this file if it's not useful

File: app/helpers/relationships_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module RelationshipsHelper
```
Comment (huongnt-2545) [inline app/helpers/relationships_helper.rb:1]: delete this file

File: app/views/users/show_follow.html.erb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+% provide(:title, @title) %>
```
Comment (huongnt-2545) [inline app/views/users/show_follow.html.erb:1]:
```suggestion
<% provide(:title, @title) %>
```

File: test/controllers/relationships_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (huongnt-2545) [inline test/controllers/relationships_controller_test.rb:1]: delete this file

**Comment by balqis277** [comment]
ready

**Comment by balqis277** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged
You can merge this pull

**Pull request merged**